app/services/rss_reader/assembler.rb

```diff
@@ -96,10 +96,10 @@ def parse_and_translate_tweet!(html_doc)
         bq_with_p = bq.css("p")
         next if bq_with_p.empty?
 
-        second_content = bq_with_p.css("p")[1].css("a")[0].attributes["href"].value
-        if bq_with_p.length == 2 && second_content.include?("twitter.com")
+        if (tweet_link = bq_with_p.css("a[href*='twitter.com']"))
```

Comment:
Changed the selector to match Twitter URLs instead of doing that separately with `.include?`.